sites/teams.html

```diff
@@ -25,9 +25,9 @@ <h1 class="main-header">
 </nav>
 <section class="teams-selection-section">
     <div class="teams-selection">
-        <div class="team">
+        <div class="team home">
```

Comment:
to be more consistent of BEM you should write this class like team-selection__team team-selection__team--home